public/app/features/alerting/partials/notification_edit.html

```diff
@@ -32,6 +32,16 @@ <h3 class="page-sub-heading" ng-show="ctrl.isNew">New Notification Channel</h3>
           checked="ctrl.model.settings.uploadImage"
           tooltip="Captures an image and include it in the notification">
       </gf-form-switch>
+      <div class="grafana-info-box m-t-2" ng-show="ctrl.model.settings.uploadImage && !ctrl.rendererAvailable">
+        <h5>No image renderer available/installed</h5>
+        <p>
+          For Grafana to be able to capture an image and include it in the notification you need to install the
+          <a href="https://grafana.com/grafana/plugins/grafana-image-renderer" target="_blank">Grafana Image Renderer plugin</a>.
```

Comment:
For security reasons, you [should](https://developers.google.com/web/tools/lighthouse/audits/noopener) combine `target="_blank"` with `rel="noreferrer"`:
```suggestion
          <a href="https://grafana.com/grafana/plugins/grafana-image-renderer" target="_blank" rel="noreferrer">Grafana Image Renderer plugin</a>.
```